Cargo.lock

```diff
@@ -2103,7 +2103,6 @@ dependencies = [
  "bincode",
  "blake3",
  "byteorder",
- "bytes 0.4.12",
```

Comment:
Why does this not remove the corresponding `[[package]]` section?

Reply:
There are still other dependencies that require bytes 0.4